src/main/java/com/blackducksoftware/integration/hub/cli/CLIDownloadService.java

```diff
@@ -204,6 +203,7 @@ private void updateJreSecurity(final IntLogger logger, final CLILocation cliLoca
                     .getValue(CIEnvironmentVariables.BDS_CACERTS_OVERRIDE);
             final File customCacerts = new File(customCacertsPath);
 
+            final String cacertsFilename = customCacerts.getName();
```

Comment:
This should allow Users to point to their certificate file with whatever name it is.Ex:  cacerts, jssecacerts